objectivec/GPBDescriptor.m

```diff
@@ -561,6 +567,15 @@ - (instancetype)initWithFieldDescription:(void *)description
   return self;
 }
 
+- (instancetype)initWithFieldDescription:(void *)description
```

Comment:
I don't think this is actually needed since all calls here should be from within the runtime?  If they have to copies loaded, other things are likely to go wrong.  But I guess it doesn't hurt to play it safe.